pkgs/by-name/co/committed/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+, testers
+, nix-update-script
+, committed
+}:
+let
+  version = "1.0.20";
+in
+rustPlatform.buildRustPackage {
+  pname = "committed";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "crate-ci";
+    repo = "committed";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-HqZYxV2YjnK7Q3A7B6yVFXME0oc3DZ4RfMkDGa2IQxA=";
+  };
+  cargoHash = "sha256-AmAEGVWq6KxLtiHDGIFVcoP1Wck8z+P9mnDy0SSSJNM=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
+
+  passthru = {
+    tests.version = testers.testVersion { package = committed; };
+    updateScript = nix-update-script { };
```

Comment:
Never seen this in all the packages I've upstreamed so far.
Even without it, projects of mine such as `ttfb`, get [updated automatically](https://github.com/NixOS/nixpkgs/pull/308691).

What's your intention here?

Reply:
I am not sure if this is required or not. I looked through some other packages in `pkgs/by-name` to see if I was missing anything in the package definition, and saw that some packages set this. To be on the safe side I set this as well after testing that the update script works with the package definition locally.

Is there a recommended practice regarding `passthru.updateScript`? It is mentioned in the [stdenv docs](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#passthruupdatescript-var-passthru-updatescript), but I have no preference either way.

Reply:
@astro What do you think?

Reply:
My opinion is to leave out optional clutter if the package can be updated with a plain `src` update anyway.